bitnami/redis/values.yaml

```
@@ -1194,6 +1194,9 @@ metrics:
     ## @param metrics.serviceMonitor.relabellings Metrics relabellings to add to the scrape endpoint
     ##
     relabellings: []
+    ## @param metrics.serviceMonitor.metricRelabelings Metrics metricRelabelings to add to the scrape endpoint
+    ##
+    metricRelabelings: []
```

Please also update the values `@param`:

```suggestion
    ## @param metrics.serviceMonitor.relabellings Metrics RelabelConfigs to apply to samples before scraping.
    ##
    relabellings: []
    ## @param metrics.serviceMonitor.metricRelabelings Metrics RelabelConfigs to apply to samples before ingestion.
    ##
    metricRelabelings: []
```